### PR TITLE
jira estimate UDA

### PR DESCRIPTION
**Pull request by devenv**

### CODE REVIEW

File: bugwarrior/services/jira.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class JiraIssue(Issue):
     URL = 'jiraurl'
     FOREIGN_ID = 'jiraid'
     DESCRIPTION = 'jiradescription'
+    ESTIMATE = 'estimate'
```
Comment (coddingtonbear) [inline bugwarrior/services/jira.py]: Unfortunately UDAs are global, so we should really do what we can to not pollute the user's UDAs with service-specific ones.  Could I convince you into changing this to `jiraestimate`?

File: bugwarrior/services/jira.py [hunk 2]
```diff
@@ -91,6 +97,11 @@ def get_summary(self):
             return self.record['fields']['summary']['value']
         return self.record['fields']['summary']
 
+    def get_estimate(self):
+        if self.extra.get('jira_version') == 4:
+            return self.record['fields']['timeestimate']['value']
+        return self.record['fields']['timeestimate'] / 60 / 60
```
Comment (coddingtonbear) [inline bugwarrior/services/jira.py]: Are we guaranteed to have a numeric value here? Maybe we should catch `TypeError` and `KeyError' so we can return `None` if either the value is null or that key doesn't exist.

**Comment by devenv** [comment]
Added exceptions + changed to jiraestimate :)

File: bugwarrior/docs/services/jira.rst [hunk 3]
```diff
@@ -109,3 +109,4 @@ Provided UDA Fields
 +---------------------+---------------------+---------------------+
 | ``jiraurl``         | URL                 | Text (string)       |
 +---------------------+---------------------+---------------------+
+| ``estimate``        | Estimate            | Decimal (numeric)   |
```
Comment (ralphbean) [inline bugwarrior/docs/services/jira.rst]: Ah, can you change the name here in the docs too?  One last thing! :D

**Comment by devenv** [comment]
Hehe, forgot that one :)
Done

**Comment by ralphbean** [comment]
Looks great.  Thanks!  :)

**Pull request merged**